k2/torch/csrc/deserialization.h

```diff
@@ -22,9 +22,18 @@
 #include <string>
 
 #include "k2/csrc/fsa.h"
+#include "k2/torch/csrc/utils.h"
```

Comment:
Why do we need to include this header in this file? Where is it used?

Reply:
It may need by the code I previous commited,  can remove now.